### PR TITLE
Feature/stats d integration

### PR DESCRIPTION
**Pull request by louismunro**
Just a stub for the final pull request. 

This will add StatsD integration to PF so that we can simply call e.g. $statsd->timing( $stat, $time ) to log timing information.

### CODE REVIEW

File: lib/Etsy/StatsD.pm [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  CORE::send( $sock, $msg, 0 );
+}
+
+=head1 SEE ALSO
+
+L<http://codeascraft.etsy.com/2011/02/15/measure-anything-measure-everything/>
+
+=head1 AUTHOR
+
+Steve Sanbeg L<http://www.buzzfeed.com/stv>
```
Comment (extrafu) [inline lib/Etsy/StatsD.pm:136]: What's the copyright and license of that file/code?

File: lib/pf/StatsD.pm [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    initStatsd;
+}
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2014 Inverse inc.
```
Comment (extrafu) [inline lib/pf/StatsD.pm]: 2015

File: lib/Etsy/StatsD.pm (hunk 1 above)
Comment (louismunro) [inline lib/Etsy/StatsD.pm:136]: Good point.

License is extremely liberal.
https://github.com/etsy/statsd/blob/master/LICENSE

Not sure if that's BSD or MIT.

Comment (extrafu) [inline lib/Etsy/StatsD.pm:136]: Perfect, please include it in the file itself.

Thanks!

**Comment by julsemaan** [comment]
Will that be in PF5, if yes then I guess an associated issue should be opened and the proper milestone associated to it.

This also needs rebasing

**Comment by louismunro** [comment]
It will be in 5.0.
Do we really want to use use issues for new features?
I thought it was for bugs/improvements.

**Comment by julsemaan** [comment]
I'd put it in issues to track it outside of the code. Not absolutelly necessary but it gives us a central point to see the projects and in which version they belong

**Comment by louismunro** [comment]
I am closing this one.
Will reopen another as the branches had diverged too much to my taste and so I created a new feature branch rather than go to the trouble to rebase.

**Pull request closed without merging**